app/views/teams/index.html.slim

```diff
@@ -2,12 +2,28 @@ nav.actions
   ul.list-inline
     li = link_to 'My Team', current_student.current_team, class: 'btn btn-default btn-sm' if current_student.current_team
     li = link_to icon('plus', 'New Team'), new_team_path, class: 'btn btn-primary btn-sm' if can? :create, Team.new(season: Season.current)
-  div.pull-right.dropdown
-    button.dropdown-toggle.form-control  data-toggle="dropdown"
+  div.pull-right.dropdown.padding-dropdown
+    button.btn.btn-default.dropdown-toggle data-toggle="dropdown" aria-haspopup="true" 
```

Comment:
👍 
Just curious: where does the `aria` attribute come from and what does it mean? We don't have a popup here, that's why I'm wondering... But I'm also not familiar with these tags  🐣